### DISCUSSION TITLE
Is there an API LIMIT in place?

### DISCUSSION THREAD

**Discussion by adambirds** [lukePeavey/quotable#113] [Questions] [answered]
Is there an API LIMIT in place? If so, what is that limit?

**Comment by lukePeavey** [comment, marked as answer]
**No, there are no limits on the number of API requests you can make**

Some of the endpoint do limit the maximum number of results you can get per page. This is specified in the docs (usually the max is 150).

**Reply by adambirds** [reply]
Okay cool, just wanted to see if I needed to handle an api limit on the requests. Thanks for the quick response. We're only using the random endpoint so only want 1 result at a time anyway. Thanks.

**Comment by OpenSourceSimon** [comment]
There's a rate limit now btw. Look at #165 for more information

**Comment by lukePeavey** [comment]
Yes, rate limit is now 200 requests per minute, by up address
